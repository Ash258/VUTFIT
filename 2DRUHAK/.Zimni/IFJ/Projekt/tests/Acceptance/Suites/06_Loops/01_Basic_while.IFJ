' Basic while loop

scope
	dim a as double = 25.54
	do while a > (0 - 25)
		print a;
		a = a - 3.29
	loop
end scope
