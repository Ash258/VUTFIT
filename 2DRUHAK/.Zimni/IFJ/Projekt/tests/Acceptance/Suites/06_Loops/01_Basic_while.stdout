 25.54 22.25 18.96 15.67 12.38 9.09 5.8 2.51-0.78-4.07-7.36-10.65-13.94-17.23-20.52-23.81